modules/log-collector-rsyslog-server.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-rsyslog-server_{context}"]
+= Configuring the collector to listen for connections as an syslog server
+
+You can configure your log collector to collect logs from outside the cluster by specifying `syslog` as a receiver input in the `ClusterLogForwarder` custom resource (CR). This enables you to use a single common log store for logs that are collected from both inside and outside of your {product-title} cluster.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Modify the `ClusterLogForwarder` CR to add configuration for the `syslog` receiver input:
++
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1beta1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  serviceAccountName: <service_account_name>
+  inputs:
+    - name: syslog-receiver
+      receiver:
+        type: syslog
+        syslog:
+          port: 10514 # <1>
+          protocol: tcp # <2>
+  pipelines:
+    - name: syslog-pipeline
+      inputRefs:
+        - syslog-receiver
+----
+<1> Specify the port that the input receiver listens on. This must be a value between `1024` and `65535`. The default value is `10514`.
+<2> Specify the protocol for the connection that the input receiver listens on. This can be `tcp` or `upd`.
```

Comment:
probably not needed?

Reply:
No, this is not needed anymore, as we are not giving the choice to the user and we are always using tcp to support tls.